spec/rubocop/cop/rspec/verified_double_reference_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'verified double references' do |verified_double|
```

Comment:
Cosmetic. Is there a reason to have this shared example as a gobal, shared one? Or is it sufficient to nest it under `RSpec.describe RuboCop::Cop::RSpec::VerifiedDoubleReference do`?

Reply:
Handled by inlining the shared examples